azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/features/SettingsFeatures.kt

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
We can follow same pattern as other files
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
Changed as suggested.